README.md

```diff
@@ -29,7 +29,7 @@ This repo consists of the Backend code of the project, the backend code is in [S
 * `.NET` as well
 
 ## Contributing
-* Go to `Contributing.md`
+* Go to [ontributing.md](./CONTRIBUTING.md)
```

Comment:
There's a small typo here, it should be "Contributing". You missed the 'C'

Reply:
Sorry will fix that